src/ds/ds5/ds5-device.cpp

```diff
@@ -692,6 +695,27 @@ namespace librealsense
                 std::make_shared<thermal_compensation>(_thermal_monitor,thermal_compensation_toggle));
             }
 
+            auto const CUR_GVD_VERSION = gvd_buff[gvd_version_offset];
+            auto const MIN_GVD_VERSION = 0x4;
```

Comment:
Lets create a function for that, maybe inside ds5-private.h

This function will check once that `MIN_GVD_VERSION <= CUR_GVD_VERSION`
You also need to add comments for it.

Reply:
Please inside the function add comments specificaly say how we change the name e.g. from D435i to D435IF
So when we search the code in the future for D435IF we will find it

Reply:
> Lets create a function for that, maybe inside ds5-private.h
> 
> This function will check once that `MIN_GVD_VERSION <= CUR_GVD_VERSION` You also need to add comments for it.

Due to multiple definitions, the project failed on the build with my function in *.h file.
What about creating a function in ds5-device.cpp?

Reply:
Resolved together